### PR TITLE
Updated the README file.

### PR DESCRIPTION
**Pull request by nishith-pubnub**

### CODE REVIEW

**Comment by polarweasel** [comment]
Heading levels should be consecutive; don't skip levels like you've done in the Reference and Resources sections.

**Comment by nishith-pubnub** [comment]
Fixed the hierarchy. Jeff will review and make further changes to finish this task.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 
 [![](https://data.jsdelivr.com/v1/package/npm/chat-engine/badge)](https://www.jsdelivr.com/package/npm/chat-engine)
 [![Build Status](https://travis-ci.org/pubnub/chat-engine.svg?branch=master)](https://travis-ci.org/pubnub/chat-engine)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/d8b6e41d61164170873bb8fe79bab020)](https://www.codacy.com/app/PubNub/chat-engine?utm_source=github.com&utm_medium=referral&utm_content=pubnub/chat-engine&utm_campaign=badger)
 [![Codacy Badge](https://api.codacy.com/project/badge/Coverage/d8b6e41d61164170873bb8fe79bab020)](https://www.codacy.com/app/PubNub/chat-engine?utm_source=github.com&utm_medium=referral&utm_content=pubnub/chat-engine&utm_campaign=badger)
 
+PubNub ChatEngine is an object oriented event emitter based framework for building chat applications in Javascript. It reduces the time to build chat applications drastically and provides essential components like typing indicators, online presence monitoring and message history out of the box.
+
+The real time server component is provided by PubNub. ChatEngine is designed to be extensible and includes a plugin framework to make adding new features simple.
```
Comment (polarweasel) [inline README.md]: "realtime" or "real-time"

File: README.md [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
+
+# Documentation
+
+You can find the full docs on [the Github documentation website](https://pubnub.github.io/chat-engine/).
```
Comment (polarweasel) [inline README.md]: delete "Github"

**Pull request merged**